src/Resources/config/sylius/fixtures.yaml

```diff
@@ -5,3 +5,23 @@ sylius_fixtures:
rra ipsum nunc. Nibh mauris cursus \r\nmattis molestie a iaculis at. Faucibus pulvinar elementum integer enim. \r\nUt porttitor leo a diam sollicitudin tempor id eu.</p>\r\n<p>Nec sagittis aliquam malesuada bibendum arcu vitae elementum \r\ncurabitur. Dolor magna eget est lorem ipsum. Neque ornare aenean euismod\r\n elementum nisi quis eleifend. Volutpat sed cras ornare arcu. Cursus \r\neget nunc scelerisque viverra. Pretium quam vulputate dignissim \r\nsuspendisse in est ante in nibh. Leo vel orci porta non pulvinar. \r\nRhoncus urna neque viverra justo. Est ullamcorper eget nulla facilisi \r\netiam dignissim diam quis enim. Augue interdum velit euismod in \r\npellentesque. Elementum tempus egestas sed sed. Vel fringilla est \r\nullamcorper eget nulla facilisi. Libero justo laoreet sit amet cursus \r\nsit amet.</p>\r\n<h2>Ipsum suspendisse ultrices gravida</h2><p>Ipsum suspendisse ultrices gravida dictum fusce ut placerat orci. \r\nAdipiscing bibendum est ultricies integer. Fermentum leo vel orci porta \r\nnon. Pharetra magna ac placerat vestibulum lectus mauris ultrices eros \r\nin. Quis auctor elit sed vulputate. Quisque egestas diam in arcu cursus.\r\n Pulvinar mattis nunc sed blandit libero volutpat sed cras. Ultrices \r\nvitae auctor eu augue ut lectus arcu bibendum. Ipsum dolor sit amet \r\nconsectetur adipiscing elit pellentesque. Lectus mauris ultrices eros in\r\n cursus turpis massa. Diam maecenas ultricies mi eget mauris pharetra et\r\n ultrices. Amet porttitor eget dolor morbi non. Consectetur adipiscing \r\nelit ut aliquam purus sit amet. Enim neque volutpat ac tincidunt vitae \r\nsemper. Porta nibh venenatis cras sed felis eget. Gravida arcu ac tortor\r\n dignissim convallis aenean et tortor. Tortor consequat id porta nibh \r\nvenenatis cras sed felis eget. A diam sollicitudin tempor id. Sed \r\nelementum tempus egestas sed sed risus pretium quam vulputate. Turpis \r\negestas integer eget aliquet nibh.</p>\r\n<p>Purus gravida quis blandit turpis cursus in. Pellentesque dignissim \r\nenim sit amet venenatis urna cursus eget. In pellentesque massa placerat\r\n duis ultricies lacus. Ipsum consequat nisl vel pretium lectus quam id. \r\nRisus pretium quam vulputate dignissim suspendisse in est. Sed turpis \r\ntincidunt id aliquet risus feugiat in. Vitae ultricies leo integer \r\nmalesuada nunc vel risus. Aliquet bibendum enim facilisis gravida neque.\r\n Adipiscing enim eu turpis egestas pretium. Lacus vestibulum sed arcu \r\nnon odio euismod lacinia at. Libero justo laoreet sit amet cursus sit \r\namet dictum.</p></div>"}},{"code":"monsieurbiz.separator","data":{"hidden":false}},{"code":"monsieurbiz.quote","data":{"author":"A customer","content":"<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>"}},{"code":"monsieurbiz.separator","data":{"hidden":true}},{"code":"monsieurbiz.button","data":{"label":"An amazing web agency","link":"https://monsieurbiz.com"}}]'
```

Comment:
You should use the following notation:

```yaml
field: |
    data in any format you like
```
Having the JSON formatted could be nice :)